torch/csrc/jit/python/script_init.cpp

```diff
@@ -783,11 +786,55 @@ void initJitScriptBindings(PyObject* module) {
                   if (auto method = self.find_method(name)) {
                     return py::cast(*method);
                   }
+                  if (self.has_property(name)) {
+                    auto prop = self.get_property(name);
+                    auto getter_func = py::cast(std::get<1>(prop));
+                    auto setter_func = py::cast(std::get<2>(prop));
+                    py::object property_func =
+                        py::module::import("builtins").attr("property");
+                    py::object property =
+                        property_func(getter_func, setter_func);
+                    return property.attr("fget")();
```

Comment:
Calling from C++ into Python into C++, amazing 😍 

I know I suggested the current implementation, but looking after looking at this, we don't need to construct the property object at all. Why don't we just call `getter_func` directly? It's a `Method`, right?